r/vignettes/developing.Rmd

```diff
@@ -1,21 +1,23 @@
 ---
 title: "Arrow R Developer Guide"
-output: rmarkdown::html_vignette
+output: 
+  html_document:
+      toc: true
+      toc_depth: 2
```

Comment:
I'm curious if there's a general movement away from `rmarkdown::html_vignette`? I've always used `rmarkdown::html_vignette` because I thought it was streamlined for inclusion in the package (and pkgdown overrides this stuff anyway for the web docs).

Reply:
Oops, I chucked this in so it made it easier to preview the list of contents when I was rendering this as RMarkdown